packages/dev/docs/pages/react-spectrum/index.mdx

```diff
@@ -155,6 +156,13 @@ A React implementation of Spectrum, Adobe’s design system.
   <ActionGroup />
 </ExampleCard>
 
+<ExampleCard
+  url="https://react-spectrum.adobe.com/react-aria/FileTrigger.html"
```

Comment:
is it confusing to link to the react-aria docs cause then it says to import file trigger from 'react-aria-components' . the only place where it's known that you can import filetrigger from '@adobe/react-spectrum' are the examples in the RSP dropzone docs

Reply:
Good point, I forgot that people might want to use FileTrigger standalone. Might be worth making a copy of the docs in RSP in that case then so that it shows up in the sidebar as well